docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -3530,6 +3530,34 @@ To allow remote connections, set to `0.0.0.0`
 
 The maximum number of WebSocket connections allowed for JSON-RPC. Once this limit is reached, incoming connections are rejected. The default is 80.
 
+### `rpc-ws-max-frame-size`
+
+=== "Syntax"
+
+    ```bash
+    --rpc-ws-max-frame-size=<INTEGER>
+    ```
+
+=== "Example"
+
+    ```bash
+    --rpc-ws-max-frame-size=1024 * 1024
+    ```
+
+=== "Environment variable"
+
+    ```bash
+    BESU_RPC_WS_MAX_FRAME_SIZE=1024 * 1024
+    ```
+
+=== "Configuration file"
+
+    ```toml
+    rpc-ws-max-FRAME-SIZE=1024 * 1024
+    ```
+
+The maximum size in bytes for JSON-RPC WebSocket frames. If this limit is exceeded, the WebSocket disconnects. The default is 1024 * 1024 (or 1 MB).
```

Comment:
I don't think it's correct that if the limit is exceeded, the socket disconnects.

Reply:
@macfarla This comes straight from the [description in the code](https://github.com/hyperledger/besu/blob/main/besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java#L759).  The original issue was Besu was disconnecting from Nimbus because the default limit was smaller than what Nimbus used.

Reply:
👍 true